### DISCUSSION TITLE
FT works except it doesn’t read videos

### DISCUSSION THREAD

**Discussion by nguxx** [FreeTubeApp/FreeTube#4289] [Q&A]
Hi’
I downloaded v0.19.1 beta. 
I’m with Windows 10, my FT loads and displays channels, suscribes etc. but cannot read videos. 
The waiting cercle turns endlessly.
Firewall is desactivated.
Something else to do?
Thanks

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi can u do the following,

Open FT -> Open developer console -> Go to a video -> check if errors are showing up in the console -> screenshot console

**Reply by nguxx** [reply]
Hi,
Where is the developer console? I can’t find it in the left menu neither in the Settings

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Oh my apologies open dev console by pressing

Ctrl + shift + i

**Reply by nguxx** [reply]
Sorry for my noob question ;-)
In deed there are errors
![filius-2023-11-07_14h09-25](https://github.com/FreeTubeApp/FreeTube/assets/54742067/b3cf53df-2583-45f2-affe-ce45f5adbaa7)

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
I haven't had any luck reproducing this issue so far. Is this happening for you on all videos?

**Reply by qupear** [reply]
Hi. Having sort of same issues except that error is ERR_ABORTED 403 and then some sponsor 404 error.
![FreeTube_9OQMAcqT0p](https://github.com/FreeTubeApp/FreeTube/assets/8069685/684b2bdd-476f-4e8a-9744-b2a9b98e8316)

For now in Freetube I can search youtube but every video is not playing. VPN doesn't help. And disabling SponsorBlock doesn't help. 

In PotPlayer all video could be played normally with SponsorBlock. But Freetube is very best in terms of history tracking.

**Reply by absidue** [reply]
@qupear You have the proxy videos through Invidious setting turned on, turning it off (it's off by default) will probably work just fine and get rid of the 403 errors.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Hi @nguxx are you still running into this issue?

**Comment by EricGus** [comment]
I am getting similar behavior ,, if I switch to LEGACY FORMATS it plays but using DASH the video will play for a half second then hang / pause .. this is on Mac OS / x86 , but I have tried it with Windows 10 and still have this problem.

Video playback is choppy, plays for a few seconds, then video freezes audio heard for a few more seconds, then the entire stream pauses, spinning circle.. after a few mins of pausing and circle spinning sometime later, video resumes for a couple seconds then the cycle repeats.. basically unwatchable.

Happens on every video at every quality stream IF USING DASH .. legacy works and seems to play ok but never goes past 720P

<img width="916" alt="image" src="https://github.com/FreeTubeApp/FreeTube/assets/5209904/4e2d098b-d491-4579-bc7e-12333f911ef5">

<img width="1345" alt="image" src="https://github.com/FreeTubeApp/FreeTube/assets/5209904/d70c11f4-2e9b-4c0d-9ddf-e188b546ae59">

**Reply by EricGus** [reply]
I should add I dont have proxy enabled this is using the LOCAL API, and no proxy.

using the latest version of FreeTube .. I should note it was working fine till a couple days ago when it just stopped (I strongly suspect it was a change on the youtube side as I tried a clean/fresh install on a different {windows 10} machine and got the exact same behavior with the install defaults}


<img width="1405" alt="image" src="https://github.com/FreeTubeApp/FreeTube/assets/5209904/c0c3412b-d51a-4d70-b153-bee109f8c45f">

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
@EricGus are you still running into this issue or is it already resolved?

**Reply by EricGus** [reply]
It comes and goes sporadically .. but I believe it might be YouTube messing with intentionally degrading performance deliberately on purpose as recently reported in the various news outlets.. eg intentionally causing performance problems for non chrome browsers / people who run adblockers. ..  but yes I still get it, but just as randomly as it happens, it also just eventually goes away usually in 3 or 4 days .. I am normally much more detailed and try to be as specific when reporting issues with software but in this case, I am not really changing anything on my end and this issue comes/goes randomly .. but as I said above my suspicions are its tided to intentional service degradation by youtube themselves on their end.  If there is anything you want me to try to capture or get logs when the issue occurs id be happy to, just explain what it is you are looking for and how I would go about providing that information.  At the moment its working but, that could change without warning at any time as I have seen..

**Comment by absidue** [comment]
@EricGus as indicated by the @ name, that message was only intended for qupear, because it solves their specific problem (the URLs in their logs include the hostname of an Invidious instance instead of googlevideo, 403s + Invidious instance hostname means YouTube has IP blocked proxying on that instance or it's a music video).

**Reply by EricGus** [reply]
Yea I did see that but I wanted to add that update to my post just in case you asked .. figured id just post it if it came up. sorry for not clarifying that.  But as I mentioned originally in my case im not using invidious and getting the same kind of behaviors on all videos when DASH is the default, "LEGACY" video setting seems to play fine.

**Comment by qupear** [comment]
I changed FT player setting to DASH formats and Voilà it plays videos now. So maybe it will make sense in case of network errors try to automatically switch player formats in FT code.

Oh, nevermind. With dash it plays videos but as @EricGus mentioned it is choppy and freezes not suitable to watch at all.

@absidue btw in general settings I have Local API not invidous if you meant that.

**Reply by qupear** [reply]
On a fresh clean Windows I've installed Freetube and everything works like a charm. IDK what cause FT to break when there is a lot of history and playlisted videos. Or maybe some settings or disk caching. Or even some networking issue on invidous or youtube side. Will try to keep FT clean from huge data maybe that way it would last longer.
